### DISCUSSION TITLE
Android App (3.110.0) weird behavior with newline

### DISCUSSION THREAD

**Discussion by abelliae** [standardnotes/forum#2913] [General]
**Describe the bug**
When pressing the 'enter' key on the Android keyboard a new empty line is added below the current line, but the cursor doesn´t go to the new line.

**To Reproduce**
1. Open a note
3. Move the cursor to the end of a line with text
4. Press enter
5. Cursor won't go to newline, but a new empty line is added below.

**Expected behavior**
A new empty line is added and the cursor moves to it.

**Smartphone (please complete the following information):**
 - Device: Pixel 5
 - OS: Android 13
 - App Version 3.110.0

**Comment by effieeee** [comment]
Hi @abelliae, may I know if this still happens with v3.114 on Android? Also, does this happen with a specific note type, apart from Plain Text?

**Comment by abelliae** [comment]
Yep, same behavior. I've only tried it in a Plain Text note.

After upgrading to v3.117, I tried with a rich text note, it doesn't seem to happen, though it still happens for plain text ones.

**Comment by effieeee** [comment]
Thanks for trying it out on the new version as well! To ensure that the mobile app now starts without any previous versions of app data, can I ask you to try following the reset steps for Android in [this link](https://docs.standardnotes.com/troubleshooting/reset-apps/#android)?

**Comment by abelliae** [comment]
Did the procedure. Still happens. I also noted it happens with rich text notes too.

**Comment by karan** [comment]
Just to add - this happens to me as well. The only way to "fix" this is to switch the note to plain text.

**Comment by myreli** [comment]
Hi @karan, in which editor are you facing this issue?

**Reply by karan** [reply]
Markdown and Minimal markdown

**Reply by myreli** [reply]
Thank you for your reply. 

I'm trying to understand what is the underlying issue as it happens on specific devices and editors.
